### DISCUSSION TITLE
Training config selection

### DISCUSSION THREAD

**Discussion by korneliaWatson** [open-mmlab/mmpose#2611] [Q&A] [answered]
Hi all, 

I'm training MMPose on a custom dataset.

I based my training configs on several configs already available in the MMPose's repo. Is there a recommended way in which I can set off a script that trains models using each customised configuration that doesn't end up using the whole memory space because of generated files? I.e. is there a way to only collect some metrics that I can later use to decide which configuration performed best in my use case?

Another question relates to image sizes. Some configs are created for image sizes of 256x192, others for 384x288. Is there a general recommendation for when a given variation should be used?

**Comment by Ben-Louis** [comment, marked as answer]
1) Regarding the selection of config, there is no pre-made script available for training and choosing the best config. It may be necessary for you to write it yourself. Additionally, the test accuracy of the model will be saved in the folder that you designated for the `--work-dir` argument.

2) Generally, a larger input size will result in better performance but will take longer for inference. It is important to select the appropriate input size based on your specific requirements.
